conmon/conmon.c

```diff
@@ -632,6 +636,45 @@ int main(int argc, char *argv[])
 	cpid = atoi(contents);
 	ninfo("container PID: %d", cpid);
 
+	/* Setup endpoint for attach */
+	struct sockaddr_un attach_addr = {0};
+	_cleanup_close_ int afd = -1;
+	int asfd = -1;
+
+	if (!exec) {
+		attach_addr.sun_family = AF_UNIX;
+		snprintf(attach_sock_path, PATH_MAX, "/var/run/%s-attach", cid);
```

Comment:
Use the symlink hack to connect to it? (create symlink to real location, connect to symlink path)